packages/react-components/react-dialog/src/components/DialogSurface/useDialogSurface.ts

```diff
@@ -50,7 +45,7 @@ export const useDialogSurface_unstable = (
     }
   });
 
-  const handleKeyDown = useEventCallback((event: React.KeyboardEvent<DialogSurfaceElementIntersection>) => {
+  const handleKeyDown = useEventCallback((event: React.KeyboardEvent<HTMLDivElement>) => {
```

Comment:
Should it be just `HTMLElement`?

Reply:
The actual internal type is `HTMLDivElement`. I'm making it generic externally (`HTMLElement`) just to allow proper casting on cases such as https://github.com/microsoft/fluentui/issues/27281